packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -3080,6 +3117,17 @@ export function attach(
       canEditFunctionPropsRenamePaths:
         typeof overridePropsRenamePath === 'function',
 
+      canToggleError:
+        supportsTogglingError &&
+        // If it's showing the real content, we can always flip it into an
+        // error state.
+        (!isErrored ||
+          // If it's showing an error state because we previously forced it to,
+          // allow toggling it back to remove the error boundary.
+          forceErrorForFiberIDs.get(id) === true),
+      // Is this error boundary in error state.
+      isErrored,
```

Comment:
We could add the `nearestErrorBoundaryID` to this message instead, and avoid having to eagerly send an `isErrorBoundary` bit over for every Fiber in the tree.